### ISSUE TITLE
FYI:  Closing Issues via Commit Messages 

### ISSUE THREAD

**Issue by tilboerner** [devsnd/cherrymusic#222]
From the github blog:

> Recently we changed the way closing issues via commit message works on GitHub.
> 
> Now when you enter "Fixes #33" in a commit message, issue 33 will only be closed once the commit is merged into your default branch (usually master). [more...](https://github.com/blog/1386-closing-issues-via-commit-messages)

Everybody's listening to the github [RSS feed](https://github.com/blog.atom), right? ;)

ps. This issue auto-closes when y'all had a look. :bomb:
